### PR TITLE
User story

### PR DESCRIPTION
**Pull request by PariplabPal11**
Message queue

### CODE REVIEW

File: .idea/settings/config/options/github_settings.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (kupaul) [inline .idea/settings/config/options/github_settings.xml:1]: Remove this file from Commit

File: .idea/settings/config/options/vcs.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (kupaul) [inline .idea/settings/config/options/vcs.xml:1]: Remove this file from Commit

File: modules/configuration/config/messaging/messaging-config.xml [hunk 3]
```diff
@@ -83,5 +83,10 @@
       <event
         name="JobPurged"/>
     </destination>
+    <destination
+      id="89"
+      internal="true"
+      name="Twilio Message Sending"
```
Comment (kupaul) [inline modules/configuration/config/messaging/messaging-config.xml:89]: Rename this to TwilioMessageSend

Comment (PariplabPal11) [reply modules/configuration/config/messaging/messaging-config.xml:89]: ok

**Comment by kupaul** [comment]
vbclose

**Pull request closed without merging**